helm/cluster-autoscaler-app/values.yaml

```diff
@@ -8,17 +8,32 @@ groupID: 65534
 port: 8085
 
 # Make sure to update the docs at https://github.com/giantswarm/docs/blob/master/src/layouts/shortcodes/autoscaler_utilization_threshold.html if you change the defaults below
-configmap:
-  expander: "least-waste"
-  scanInterval: "10s"
-  scaleDownUtilizationThreshold: 0.65
-  skipNodesWithLocalStorage: "false"
-  skipNodesWithSystemPods: "true"
-
 image:
   registry: quay.io
   name: giantswarm/cluster-autoscaler
   tag: v1.15.2
 
-clusterID:
-  "test-cluster"
+clusterID: "test-cluster"
+provider: "changeme"
+
+azure:
+  subscriptionID: "changeme"
```

Comment:
If this is going to be set by the cluster values configmap please comment it like @corest has done for Ingress Controller.

https://github.com/giantswarm/nginx-ingress-controller-app/blob/963189b8e0981e65810e41a65d027c65f64e948b/helm/nginx-ingress-controller-app/values.yaml#L12-L14